rllib/examples/rock_paper_scissors_multiagent.py

```diff
@@ -14,7 +14,7 @@
 from ray import tune
 from ray.rllib.agents.pg.pg import PGTrainer
 from ray.rllib.agents.pg.pg_tf_policy import PGTFPolicy
-from ray.rllib.policy.policy import Policy
+from ray.rllib.policy.tests.test_policy import TestPolicy
```

Comment:
I would prefer to directly extend Policy, since it means less abstractions for a new user to understand.

Reply:
Sure ..

Reply:
done.